### ISSUE TITLE
The latest GPT4 Pass@1 is around 85% on HumanEval.

### ISSUE THREAD

**Issue by ChiYeungLaw** [SLAM-group/newhope#5]
We should not use the old score of GPT4 (67) in the leaderboard.

**Comment by BowieHsu** [comment]
@ChiYeungLaw It looks like humaneval is about to become the MNIST of the LLM era.

**Comment by ChiYeungLaw** [comment]
> @ChiYeungLaw It looks like humaneval is about to become the MNIST of the LLM era.

hahaha, I think so, but the Humaneval only considers Python algorithm-related problems. Evaluated with more diverse coding problems like DS-1000 is needed.

**Comment by jamesmurdza** [comment]
Agreed. Here are some more updated results from HumanEval: https://github.com/jamesmurdza/humaneval-results
